waku/protocol/v2/waku_store.nim

```diff
@@ -292,10 +292,20 @@ method init*(ws: WakuStore) =
   ws.handler = handle
   ws.codec = WakuStoreCodec
 
-proc init*(T: type WakuStore, switch: Switch, rng: ref BrHmacDrbgContext): T =
+  if ws.store.isNil:
```

Comment:
Is there a eason to do it this way and not be more explicit with `if ws.store`/flag then <store path>?

Reply:
I mean adding a flag is just another thing, we only initizalize the store right now if a dbpath is set. I think that is explicit.

Reply:
aight